parlai/crowdsourcing/tasks/model_chat/impl.py

```diff
@@ -15,10 +15,9 @@
 from parlai.crowdsourcing.tasks.model_chat.model_chat_blueprint import (
     SharedModelChatTaskState,
 )
-import parlai.crowdsourcing.tasks.model_chat.worlds as world_module
 
 
-def run_task(cfg: DictConfig, task_directory: str):
+def run_task(cfg: DictConfig, task_directory: str, world_module=None):
```

Comment:
adding `world_module` here so it doesn't have to call the `model_chat.worlds` by default but any `world_module` as long as it's consistent with `cfg.mephisto.blueprint.world_file`.